### DISCUSSION TITLE
Unit laravel

### DISCUSSION THREAD

**Discussion by LfJohnVo** [serversideup/docker-php#347] [Q&A]
"Is there a step-by-step guide on how to use one of your images with an implementation of Nginx Unit for Laravel? No matter how much I try, I can't get it to work, and I would greatly appreciate it."

**Comment by jaydrogers** [comment]
# ✅ Huge Update
In v4, we redid the entire docs site. We now have a "[Quick Start](https://serversideup.net/open-source/docker-php/docs/getting-started/installation)" example as well as a guide on how to go from [Development to Production](https://serversideup.net/open-source/docker-php/docs/deployment-and-production/development-to-production) 🚀

Unit is now deprecated, but we have examples for FrankenPHP 😃
